build-support/docker/tc_remote_execution/Dockerfile

```diff
@@ -55,3 +55,13 @@ ENV PATH "${PYENV_ROOT}/versions/${PYTHON_27_VERSION}/bin:${PATH}"
 ENV PATH "${PYENV_ROOT}/versions/${PYTHON_36_VERSION}/bin:${PATH}"
 ENV PATH "${PYENV_ROOT}/versions/${PYTHON_37_VERSION}/bin:${PATH}"
 ENV PATH "${PYENV_ROOT}/versions/${PYTHON_38_VERSION}/bin:${PATH}"
+
+# Add a non-root user and configure image to use that user for running binaries.
+RUN adduser \
+    --disabled-password \
+    --disabled-login \
+    --uid 1000 \
+    --shell /bin/bash \
+    --gecos 'Pants User' \
+    pants
+USER pants
```

Comment:
Nit: I recommend `pantsbuild` so that it doesn't get confusing in other places we use the name `pants`, like the `./pants` script.

Reply:
How about `pantsuser` since `pantsbuild` is the GitHub org name?

Reply:
That's fine too. Typically we use `pantsbuild` whenever we need something like a user name, but `pantsuser` works too.

Reply:
I'll use `pantsbuild` then; no need to deviate from existing convention.